docs/modules/ROOT/pages/wizard-plugin.adoc

```diff
@@ -0,0 +1,83 @@
+[[contracts-wizard-deploy-plugin]]
+= Contracts Wizard Deploy Plugin
+
+When configuring contracts from https://wizard.openzeppelin.com/[Contracts Wizard, window=_blank], you can directly deploy the configured Smart Contract using your Defender account.
+
+
+[[usage]]
+== Usage
+
+[[api-key-generation]]
+=== API Key generation
+In your Defender dashboard, go to *Settings -> API Keys* and click *Create API Key*, you only need _Manage Deployments_ permission.
+
+WARNING: We also recommend to set an expiration for the API Key, considering that is going to be used from an external site.
+
+image::remix-plugin-api-key.png[Defender Remix Plugin Api Key, 400, 400]
+
+[[deploying-from-wizard]]
+=== Deployment from Contracts Wizard
+
+Go to https://wizard.openzeppelin.com/[Contracts Wizard, window=_blank] site, and after editing your contract, click on "Deploy with Defender".
+
+TODO add image here
```

Comment:
We have TODO here

Reply:
Beat me to it :P

Reply:
> We have TODO here

I'm was trying to engage open source contributors to do it for me 😉